tests/next_tests/integration_tests/feature_tests/iterator_tests/test_horizontal_indirection.py

```diff
@@ -101,6 +102,9 @@ def test_direct_offset_for_indirection(program_processor):
     if program_processor == run_dace_iterator:
         pytest.xfail("Not supported in DaCe backend: shift offsets not literals")
 
+    if program_processor == gtfn_cpu.run_gtfn_with_temporaries:
+        pytest.xfail("Dynamic offsets not supported in temporaries pass.")
```

Comment:
what's the strategy to resolve that? I guess the pass should just not try to extract, but should work, right?

Reply:
As far as I remember we only need to extend the `TraceShifts` pass. Shouldn't be very time intensive.

Reply:
Idea: TraceShift returns something like `(SomeDimension, DynamicShiftDistance)`. Temporary pass itself should just work.

Reply:
Can you add this comment to the code.

Reply:
Thinking about this again I was wrong in assuming this would be easy to fix. I've added my reasoning as a comment in `TraceShift`.